server/models/Profile.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require('mongoose');
+
+const dayAvalSchema = new mongoose.Schema({
+    from: String,
+    to: String,
+    isAvailable: {
+        type: Boolean,
+        default: true
+    },
+    price: Number
+})
+
+const profileSchema = new mongoose.Schema({
+    user: {
+        type: mongoose.Types.ObjectId,
+        ref: "User",
+        required: true
+    },
+    firstName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    lastName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    description: String,
+    gender: {
+        type: String,
+        required: true,
+        lowercase: true,
+        trim: true,
+        enum: ['male', 'female', 'other']
+    },
+    birthDate: Date,
+    phoneNumber: {
+        type: String,
+        trim: true
+    },
+    address: {
+        type: String,
+        required: true
+    },
+    availability: {
+        0: [dayAvalSchema],
+        1: [dayAvalSchema],
+        2: [dayAvalSchema],
+        3: [dayAvalSchema],
+        4: [dayAvalSchema],
+        5: [dayAvalSchema],
+        6: [dayAvalSchema],
+    },
```

Comment:
A few concerns here:
- I would say stick with day names as keys instead of numbers for readability purposes. 
- I noticed each day has an array of availability blocks, this could work but it's a much more complicated approach. For example, did we think about sanitizing/validating the blocks in each day (e.g. cases like overlapping blocks?)
- What will the UI/UX look like for sitters setting availability? A calendar-like view where they can set up multiple blocks of availability? Are these changes talked about/planned out?

Reply:
Hey @ugur-akin, for some clarification on why Abdul chose numbers, please see below:

Abdul's message to me in the slack chat:
"Numbers here represent all days  from Sunday to Saturday. I intentionally chose to represent them as numbers here for two reasons:
1- days are represented by numbers in Javascript, so for a specific date if we go like "date.getDay()" it would return the corresponding number. So the interesting thing here is we will take that number (the day) on front-end and set its availability accordingly. Note that in this case even the run time here is constant."